editor/store/selectors.js

```diff
@@ -1285,21 +1286,30 @@ function buildInserterItemFromSharedBlock( state, enabledBlockTypes, sharedBlock
  * items (e.g. a regular block type) and dynamic items (e.g. a shared block).
  *
  * @param {Object}           state             Global application state.
- * @param {string[]|boolean} enabledBlockTypes Enabled block types, or true/false to enable/disable all types.
+ * @param {string[]|boolean} allowedBlockTypes Allowed block types, or true/false to enable/disable all types.
  *
  * @return {Editor.InserterItem[]} Items that appear in inserter.
  */
-export function getInserterItems( state, enabledBlockTypes = true ) {
-	if ( ! enabledBlockTypes ) {
+export function getInserterItems( state, allowedBlockTypes ) {
+	if ( allowedBlockTypes === undefined ) {
+		allowedBlockTypes = true;
+		deprecated( 'getInserterItems with no allowedBlockTypes argument', {
```

Comment:
Both depreciation messages should be also added to https://github.com/WordPress/gutenberg/blob/master/docs/deprecated.md for better visibility.